lib/core/form/components/widgets/container/container.widget.ts

```diff
@@ -65,23 +69,44 @@ export class ContainerWidgetComponent extends WidgetComponent implements OnInit,
      */
     get fields(): FormFieldModel[] {
         const fields = [];
+        const toBeComputed = [];
```

Comment:
do we have unit test for this part ?

Reply:
Yes, the one called `should serializes the content fields with rowspan` in `container.widget.spec.ts` 👍